apps/token/src/i18n/translations/dev.json

```diff
@@ -633,5 +633,7 @@
   "MinProposalRequirements": "You must have at least 1 VEGA associated to make a proposal",
   "totalSupply": "Total Supply",
   "viaWallet": "via wallet",
-  "viaContract": "via vesting"
+  "viaContract": "via vesting",
+  "ProposalNotFound": "Proposal not found",
+  "ProposalNotFoundDetails": "The proposal you are looking for is not here, it may have been enacted before the last chain re-store. You could check forums/discord instead."
```

Comment:
```suggestion
  "ProposalNotFoundDetails": "The proposal you are looking for is not here, it may have been enacted before the last chain restore. You could check the Vega forums/discord instead for information about it."
```

Reply:
non-blocking copy change